sklearnex/covariance/incremental_covariance.py

```diff
@@ -197,6 +199,23 @@ def _onedal_partial_fit(self, X, queue=None, check_input=True):
 
         return self
 
+    @wrap_output_data
+    def score(self, X_test, y=None):
+        est = clone(self)
+        est.set_params(**{"assume_centered": True})
+
+        xp, _ = get_namespace(X_test)
+
+        location = self.location_
+        if "numpy" not in str(xp).lower():
+            location = xp.asarray(location, device=X_test.device)
+
+        # test_cov is a numpy array, but calculated on device
```

Comment:
Let's skip integration of this part for now.
For not duplication the work, it make sense to us first finalize #1813 and #1861